File: ruby/main2.rb
class Base
end

start = Time.now

for i in 0..100000000
  obj = Base.new()
end

finish = Time.now

elapsed = ((finish - start) * 1000).round

File.open('output.txt', 'a') do |fout|
  fout.puts "#{elapsed}\n"
end

File: javascript/main2.js
class Base {}

const fs = require('fs');
const path = require('path');

const start = Date.now();

for (let i = 0; i < 100000000; ++i) {
    obj = new Base();
}

const finish = Date.now();
    
const elapsed = finish - start;

const fout = path.join(__dirname, 'output.txt');
fs.appendFile(fout, elapsed + "\n", 'utf8', (err) => {
    if (err) {
        console.error('Can\'t write to a file:', err);
    }
});
